src/Gui/Viewer/CameraManipulator.hpp

```diff
@@ -69,7 +69,8 @@ class RA_GUI_API CameraManipulator : public QObject
     virtual bool handleKeyPressEvent( QKeyEvent* event,
                                       const KeyMappingManager::KeyMappingAction& action ) = 0;
```

Comment:
could we provide a default implementation "return false" so that it's not necessary to make derived class handle this ?
(it can be considered out of scope of this PR, but it would reduce code duplication).

Reply:
sure, for both `handleKeyPressEvent` and `handleKeyReleaseEvent`.